main.go

```diff
@@ -86,6 +85,18 @@ func main() {
 	// Initialized developer mode flag and log level for the logger
 	log.InitializeLogger(config.IsLogJSON(), config.GetLogLevel())
 
+	// Initialize sentry client
+	haltSentry, err := sentry.InitializeSentryClient(
+		sentry.WithRelease(controller.Commit),
+		sentry.WithEnvironment(config.GetEnvironment()),
+	)
+	if err != nil {
+		log.Panic(nil, map[string]interface{}{
+			"err": err,
+		}, "failed to setup the sentry client")
+	}
+	defer haltSentry()
```

Comment:
I may read it wrong, but why do you want to defer the call to `sentry.InitializeSentryClient()` ? I can understand that you want to defer the call to stop it when the program exits, but not its init... Or did I miss something ?

Reply:
`InitializeSentryClient` provides a mechanism to stop the goroutine that is spawned for sentry service.

Reply:
ok, got it. Thanks for the clarification. The naming was a bit confusing at first sight: I was expecting a kind of `sentry client` to be returned by the `InitializeSentryClient` function, rather than a function to close it, hence my confusion ;)